### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Sahar-AbdelSamad**
In this branch I:
- Created this React app using `Create React App (CRA)`
- Added Linters files
- Fixed Linters errors
- Updated the README file

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
+export default App;
```
Comment (julie-ify) [inline src/App.js:29]: - [ ] Let's make sure that the application should run in the browser without errors. Below is the error I get on the browser from running `npm start`. Check out this [thread](https://stackoverflow.com/questions/63912721/failed-to-load-config-react-to-extend-from) on Stackoverflow to learn how to fix this error.

Loom_edcMpA15UT

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Sahar-AbdelSamad,

Good job so far!

## Highlights!
- Good feature branch name ✔️
- Gitflow ✔️  
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Sahar-AbdelSamad) [reply src/App.js:29]: Thank you :+1: I fixed it :smile:

**Review by eri8-9 — APPROVED** [review]
Hi @Sahar-AbdelSamad , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**